src/dodal/devices/areadetector/epics/pimte_controller.py

```diff
@@ -0,0 +1,65 @@
+import asyncio
+from typing import Optional, Set  # , Set
+
+from ophyd_async.core import AsyncStatus, DetectorControl, DetectorTrigger
+from ophyd_async.epics.areadetector.drivers.ad_base import (
+    DEFAULT_GOOD_STATES,
+    DetectorState,
+    start_acquiring_driver_and_ensure_status,
+)
+from ophyd_async.epics.areadetector.utils import ImageMode, stop_busy_record
+
+from dodal.devices.areadetector.epics.drivers.pimte1_driver import Pimte1Driver
+
+TRIGGER_MODE = {
+    DetectorTrigger.internal: Pimte1Driver.TriggerMode.internal,
+    DetectorTrigger.constant_gate: Pimte1Driver.TriggerMode.ext_trigger,
+    DetectorTrigger.variable_gate: Pimte1Driver.TriggerMode.ext_trigger,
+}
+
+
+class PimteController(DetectorControl):
+    def __init__(
+        self,
+        driver: Pimte1Driver,
+        good_states: Set[DetectorState] = set(DEFAULT_GOOD_STATES),
+    ) -> None:
+        self.driver = driver
+        self.good_states = good_states
+
+    def get_deadtime(self, exposure: float) -> float:
+        return exposure + 0.1
```

Comment:
Is deadtime exposure + 0.1s ? or is it a flat 0.1s?

Reply:
checked the manual and change the dead time to the hardware limit.

https://www.princetoninstruments.com/wp-content/uploads/2020/04/PI-MTE3-System-Manual-Issue-1-4411-0162.pdf